packages/lerna-semantic-release-pre/index.js

```diff
@@ -89,26 +99,87 @@ function bumpVersionCommitAndTag (nextRelease, done) {
   });
 }
 
+function updateDependencies (pkg, updates, dependenciesKey) {
```

Comment:
After writing this, I thought that this feature probably only makes sense when you use `lerna bootstrap` otherwise, this might be an undesired behavior and actually cause some bugs, because you might not test the package `b` with the new version of `a`.

I will probably close this PR, 'cause it appears to be a dangerous behavior